### ISSUE TITLE
Phenotype completeness report

### ISSUE THREAD

**Issue by kbab** [mpi2/PhenotypeData#633]
The Phenotype Completeness report is needed for a variety of reasons. Normally MR ran these. Now that he has left we need to ensure we can continue running it.

The aim is to find the documentation related to the report and ensure that we can run it.

**Comment by kbab** [comment]
I am not sure how this report was run in the past, but the code is in the PhenotypeData git repository: PhenotypeData/reports/src/main/java/org/mousephenotype/cda/reports2/ProcedureCompletenessAndPhenotypeHits.java

I have managed to run it from my local version using intellij. I had to:

1) set an environment variable for the solr repository: internal_solr_url=http://ves-pg-d8.ebi.ac.uk:8986/solr
2) pass a parameter for the target directory to store the report e.g. : --targetDirectory=/home/kola/temp/procreport

I looked in Jenkins but could not find the job that runs the report. Someone else may know, but at least we can run the report once the solr cores are generated, and we can set up a Jenkins job if one does not already exist.
